deployment/README.md

```diff
@@ -39,12 +39,20 @@ Stack launching is managed with `mmw_stack.py`. This command provides an interfa
 
 Before launching the Model My Watershed stack, AMIs for each service need to be generated:
 
-
 ```bash
 $ ./mmw_stack.py create-ami --aws-profile mmw-stg --mmw-profile staging \
                             --machine-type mmw-{app,monitoring,tiler,worker}
 ```
 
+### Pruning AMIs
+
+After creating several AMIs, older ones become stale and are no longer needed. To prune them use:
+
+```bash
+$ ./mmw_stack.py prune-ami --aws-profile mmw-stg --mmw-profile staging \
+                           --keep 5 --service {Application,Monitoring,Tiler,Worker}
```

Comment:
For `create-ami` we specify `--machine-type=mmw-app` but for `prune-ami` we specify `--service=Application`. Any to way to make it the same for both, or is it not worth it?